.gitignore

```diff
@@ -37,3 +37,25 @@ target
 # Ballerina
 velocity.log*
 *Ballerina.lock
+
+# VSCode
+.project
```

Comment:
Isn't `**/*.project` work here? Seems redundant to add each and every file in the project.

Reply:
Changed along with the other directories by generalizing.